File: Strings/Remove Duplicates.py
class Solution:
	def removeDups(self, S):
		# code here
		hash1 = [0]*26
		s = ''
		for item in S:
		    if hash1[ord(item)-97] == 0:
		        s+=item
		        hash1[ord(item)-97] += 1
		return s

File: Strings/Remove Duplicates.cpp
class Solution{
public:	
		
	string removeDups(string S) 
	{
	    // Your code goes here
	    int arr[26] = {0};
	    string s;
	    for(int i=0;i<S.size();i++){
	        if(arr[S[i] - 'a'] == 0){
	            s+=S[i];
	            arr[S[i] - 'a']+=1;
	        }
	    }
	    return s;
	}
};
